src/client.ts

```diff
@@ -61,6 +64,14 @@ export interface ClientSettings {
    */
   introspectionEndpoint?: string;
 
+  /**
+   * Revocatopm endpoint.
+   *
+   * Required for revoking tokens. Not supported by all servers.
+   * If not provided we'll try to discover it, or otherwise default to /revoke
+   */
+  revocationEndpoint?: string;
```

Comment:
I wonder if revokeEndpoint is a bit more terse and doesn't lose meaning? It's a nit tho

Reply:
I chose revocation for two reasons: It mirrors the language used in the RFC and by extension the language generally used in the .well-known endpoint. And also, it matches the verbiage used for introspection (i.e. instrospectionEndpoint rather than introspectEndpoint). But you're the maintainer so I'll change it if you prefer.